courses/free-courses-it.md

```diff
@@ -125,6 +131,7 @@
 * [Esercitazioni di Spring Boot](https://www.youtube.com/playlist?list=PLCbSCJEIR6CpGchit9OCI6fX_qVYs78d_) - Mauro Cicolella
 * [Java EE](https://www.youtube.com/playlist?list=PLjGYWJ4Dcy-erfReHXB9Ush0cREGSmyIe) - Sonia Zorba
 * [Programmazione a Oggetti (Java)](https://www.youtube.com/playlist?list=PLUFFnpJdi99kewGZIHpCDgarZER_-J1am) - M. Torchiano, Politecnico di Torino
+* [Programmazione ad Oggetti - Java](https://www.youtube.com/playlist?list=PLhlcRDRHVUzTruZmXalUSJAK26pouP8ST) - N. Bicocchi, Università degli studi di Modena e Reggio Emilia
```

Comment:
despite what the listing above (which should be changed, we don't include creator affiliations in our listings,

Reply:
I can change all the creator affiliations from the list. (i'm new on gitHub and i don't know if i'm using it correctly)

Reply:
thanks; you don't have to do the others just the ones you've added, although maybe do the Torino one, since I've checked it.